pkg/js/compiler/compiler.go

```diff
@@ -66,12 +70,22 @@ func NewExecuteArgs() *ExecuteArgs {
 // ExecuteResult is the result of executing a script.
 type ExecuteResult map[string]interface{}
 
+func (e ExecuteResult) Map() map[string]interface{} {
```

Comment:
Missing doc.

Reply:
maybe we need a linter rule 😄

Reply:
@tarunKoyalwar  - we've already been using `golangci-lint` so we might just need to tweak its config.

Reply:
yes